File: skiCentar/skiCentar.Model/Ticket.cs
using System;

namespace skiCentar.Model
{
    public class Ticket
    {
        public int Id { get; set; }

        public int TicketTypeId { get; set; }

        public decimal TotalPrice { get; set; }

        public string? Description { get; set; }

        public DateTime ValidFrom { get; set; }

        public DateTime ValidTo { get; set; }

        public bool? Active { get; set; }

        public virtual TicketType TicketType { get; set; } = null!;
    }
}

File: skiCentar/UI/skicentar_mobile/lib/models/ticket.dart

import 'package:json_annotation/json_annotation.dart';
import 'package:skicentar_mobile/models/ticket_type.dart';

part 'ticket.g.dart';
@JsonSerializable()
class Ticket{
  int? id;
  int? ticketTypeId;
  bool? active;
  double? totalPrice;
  TicketType? ticketType;
  String? description;
  DateTime? validFrom;
  DateTime? validTo;

  Ticket({this.id,this.ticketTypeId,this.totalPrice,this.validFrom,this.validTo});

  factory Ticket.fromJson(Map<String,dynamic> json) => _$TicketFromJson(json);

  Map<String,dynamic> toJson() => _$TicketToJson(this);
}